pipeline/lookup.csv

```diff
@@ -29678,3 +29678,892 @@ brownfield-land,,local-authority-eng:BAE,BFR53,1729669
+brownfield-land,,local-authority-eng:ELM,128,1730346
+brownfield-land,,local-authority-eng:ELM,129,1730347
+brownfield-land,,local-authority-eng:ELM,130,1730348
+brownfield-land,,local-authority-eng:ELM,131,1730349
+brownfield-land,,local-authority-eng:ELM,132,1730350
+brownfield-land,,local-authority-eng:ELM,142,1730351
+brownfield-land,,local-authority-eng:ELM,133,1730352
+brownfield-land,,local-authority-eng:ELM,134,1730353
+brownfield-land,,local-authority-eng:ELM,135,1730354
+brownfield-land,,local-authority-eng:ELM,136,1730355
+brownfield-land,,local-authority-eng:ELM,146,1730356
+brownfield-land,,local-authority-eng:ELM,137,1730357
+brownfield-land,,local-authority-eng:ELM,162,1730358
+brownfield-land,,local-authority-eng:ELM,88,1730359
+brownfield-land,,local-authority-eng:ELM,108,1730360
+brownfield-land,,local-authority-eng:ELM,138,1730361
+brownfield-land,,local-authority-eng:ELM,171,1730362
+brownfield-land,,local-authority-eng:ELM,152,1730363
+brownfield-land,,local-authority-eng:ELM,141,1730364
+brownfield-land,,local-authority-eng:ELM,148,1730365
+brownfield-land,,local-authority-eng:ELM,154,1730366
+brownfield-land,,local-authority-eng:ELM,155,1730367
+brownfield-land,,local-authority-eng:ELM,161,1730368
+brownfield-land,,local-authority-eng:ELM,168,1730369
+brownfield-land,,local-authority-eng:ELM,144,1730370
+brownfield-land,,local-authority-eng:ELM,145,1730371
+brownfield-land,,local-authority-eng:ELM,149,1730372
+brownfield-land,,local-authority-eng:ELM,159,1730373
+brownfield-land,,local-authority-eng:ELM,160,1730374
+brownfield-land,,local-authority-eng:ELM,165,1730375
+brownfield-land,,local-authority-eng:ELM,116,1730376
+brownfield-land,,local-authority-eng:ELM,153,1730377
+brownfield-land,,local-authority-eng:ELM,156,1730378
+brownfield-land,,local-authority-eng:ELM,157,1730379
+brownfield-land,,local-authority-eng:ELM,158,1730380
+brownfield-land,,local-authority-eng:EAT,BLR-3-8,1730381
+brownfield-land,,local-authority-eng:EAT,BLR-4-9,1730382
+brownfield-land,,local-authority-eng:EAT,BLR-4-10,1730383
+brownfield-land,,local-authority-eng:EAT,BLR-4-11,1730384
+brownfield-land,,local-authority-eng:EAT,BLR-4-12,1730385
+brownfield-land,,local-authority-eng:EAT,BLR-4-15,1730386
+brownfield-land,,local-authority-eng:EAT,BLR-4-16,1730387
+brownfield-land,,local-authority-eng:EAT,BLR-4-17,1730388
+brownfield-land,,local-authority-eng:EAT,BLR-7-3,1730389
+brownfield-land,,local-authority-eng:EAT,BLR-9-5,1730390
+brownfield-land,,local-authority-eng:EAT,BLR-11-1,1730391
+brownfield-land,,local-authority-eng:EAT,BLR-12-1,1730392
+brownfield-land,,local-authority-eng:EAT,BLR-3-7,1730393
+brownfield-land,,local-authority-eng:EAT,BLR-4-13,1730394
+brownfield-land,,local-authority-eng:EAT,BLR-4-14,1730395
+brownfield-land,,local-authority-eng:EAT,BLR-10-1,1730396
+brownfield-land,,local-authority-eng:EDE,BL39,1730397
+brownfield-land,,local-authority-eng:EDE,BL40,1730398
+brownfield-land,,,BLR0302,1730399

```

Comment:
No organisations

Reply:
I've removed entries with no organisation, and quoted a couple more referencence fields containing commas.